htdocs/fourn/commande/script/repair_llx_commande_fournisseur_dispatch_3.6-9.0.php

```diff
@@ -0,0 +1,160 @@
+<?php
```

Comment:
Adding maintenance script at same level than application pages is not a good idea.
Can you report this into the dedicated script    /install/repair.php  that is already protected against bad use ?

Reply:
Thank you, I have now moved the code to `install/repair.php`.